plugins/frontend/backstage-plugin-bitbucket-pullrequest/src/components/PullRequestList.tsx

```diff
@@ -0,0 +1,152 @@
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import CancelIcon from '@material-ui/icons/Cancel';
+import HourglassEmptyIcon from '@material-ui/icons/HourglassEmpty';
+import StatusFilter from '../components/StatusFilter';
+import { useEntity } from '@backstage/plugin-catalog-react';
+
+const PullRequestList: React.FC = () => {
+  const [pullRequests, setPullRequests] = useState<PullRequest[]>([]);
+  const [stateFilter, setStateFilter] = useState<string>('All');
+  const { entity } = useEntity();
+  const PROJECT = isBITBUCKETSlugSet(entity);
+  const bitbucketApi = useApi(bitbucketApiRef);
+  const projectName = PROJECT.split('/')[0];
+  const repoName = PROJECT.split('/')[1];
+  const fetchpullreq = async () => {
+    await bitbucketApi
+      .fetchPullRequestList(
+        projectName,
+        repoName,
+        stateFilter !== 'All' ? stateFilter : undefined,
+      )
+      .then(data => setPullRequests(data))
+      .catch(error => error);
+  };
+  useEffect(() => {
+    fetchpullreq();
+  });
+
+  const GetElapsedTime = ({ start }: { start: string }) =>
+    moment(start).fromNow();
+  const RenderStateIcon = ({ status }: { status: string }) => {
+    switch (status) {
+      case 'OPEN':
+        return <HourglassEmptyIcon color="primary" />;
+      case 'MERGED':
+        return <CheckCircleIcon style={{ color: 'green' }} />;
+      case 'DECLINED':
+        return <CancelIcon color="secondary" />;
+      default:
+        return null;
+    }
+  };
+
+  const columns: TableColumn<PullRequest>[] = [
+    {
+      title: 'ID',
+      field: 'id',
+      highlight: true,
+      width: '20%',
+      render: (row: Partial<PullRequest>) => (
+        <Box fontWeight="fontWeightBold">
+          <a href={row.url} target="_blank" rel="noopener noreferrer">
+            #{row.id}
+          </a>
+        </Box>
+      ),
+    },
+    {
+      title: 'TITLE',
+      field: 'title',
+      highlight: true,
+      width: '30%',
+      render: rowData => <Box fontWeight="fontWeightBold">{rowData.title}</Box>,
+    },
+    {
+      title: 'STATE',
+      field: 'state',
+      highlight: true,
+      width: '10%',
+      render: rowData => <RenderStateIcon status={rowData.state} />,
+    },
+    {
+      title: 'AUTHOR',
+      field: 'author',
+      highlight: true,
+      width: '20%',
+      render: (row: Partial<PullRequest>) => (
+        <Box fontWeight="fontWeightBold">{row.author}</Box>
+      ),
+    },
+
+    {
+      title: 'CREATED',
+      field: 'created_on',
+      highlight: true,
+      width: '20%',
+      render: (row: Partial<PullRequest>) => (
+        <GetElapsedTime start={row.created_on!} />
+      ),
+    },
+    {
+      title: 'LAST UPDATED',
+      field: 'updated_on',
+      highlight: true,
+      width: '20%',
+      render: rowData => <GetElapsedTime start={rowData.updated_on!} />,
+    },
+  ];
+
+  return (
+    <div>
+      <Content>
```

Comment:
I think think it might make sense to use ordinary <Grid />  elements here (or Box/div etc) and this component is already wrapped in a Component element by its parent and I think that'll add extra padding. Also using ContentHeader kind of ties the use of this to a tab where I could imagine a variation which is rendered in a card, though this is less of an issue.